src/control/logging/debug.go

```diff
@@ -50,9 +52,24 @@ type DefaultDebugLogger struct {
 // Debugf emits a formatted debug message.
 func (l *DefaultDebugLogger) Debugf(format string, args ...interface{}) {
 	depth := logOutputDepth
-	if len(args) == 0 {
-		// Adjust for the extra call to Debug()
-		depth += 1
+
+	// Adjust depth to account for any convenience wrappers. Enables
+	// printing of correct caller info.
+	pc := make([]uintptr, depth+5)
```

Comment:
What is used to determine that the size is depth+5?  Does +5 need to be configurable?

Reply:
Ok.  Thanks for the explanation.  I am fine with it as-is.  Just wanted to understand it better.